main-api/docker-entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env sh
+
+# Wait until the service is ready before continuing.
+# This is to ensure that the service is initialized before the API tries to connect.
+service_is_ready() {
+    NAME=$1
+    HOST=$2
+    DB_PORT=$3
+    # echo "Using service $NAME: $HOST:$PORT"
+    i=1
+    while ! nc -z $HOST $DB_PORT; do
+        sleep 1
+        i=$((i+1));
+        if [ $i -eq 600 ]; then
+            echo "Service $NAME '$HOST:$DB_PORT' not responding. Exiting..."
+            exit 1
+        fi;
+    done
+}
+
+service_is_ready "DATABASE" ${DATABASE_HOST} ${DATABASE_PORT}
+
+if [ "$1" = 'api' ]; then
+  OLD_PYTHONPATH=$PYTHONPATH
```

Comment:
Why do we need to change python path?

Reply:
So that alembic can find database.py containing Base. 

https://stackoverflow.com/questions/15648284/alembic-alembic-revision-says-import-error

Reply:
good